source/boards/views-and-sorting.rst

```diff
@@ -18,5 +18,10 @@ Manage views
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-There are a number of ways you can visualize your boards. The type of template you choose provides a set of view options. You can also change the way a board looks based on property, filter, or group.
+There are a number of ways you can visualize your boards amd cards. The type of template you choose provides a set of view options.
 
+You can organize and find the cards you're looking for with our filter, sort, and grouping options. From the Board header, you can quickly toggle on different properties, change the group display, set filters, and change how the cards are sorted. Combine views with filters to make unique boards for specific items. For example, you can create a **By milestone** board and then use a filter to only display items for specific milestones.
+
+You can add multiple properties to a filter to make it an "AND" statement. This means that both conditions that you've selected need to be true in order for the filtered results to display. For example, if you want to filter your view to only display cards associated with two milestones, you would add both of them to that filter. You can also add multiple values within the same property, which is an "OR" statement.
```

Comment:
@wuwinson, could you help refine this content - I don't think I've explained it well. Thank you!